docs/refs.bib

```diff
@@ -571,6 +571,15 @@ @misc{Hall_2006_Indecomposable
   primaryClass={quant-ph}
 }
 
+@misc{hoffman2011complexequiangulartightframes,
```

Comment:
Please refer to the way in which other bibtex keys in the rest of this document are formatted and conform to that convention, please.